Plots/Contours/NCL_ce_3_2_lg.py

```diff
@@ -0,0 +1,57 @@
+"""
```

Comment:
Review comments for above function `Plots/Contours/NCL_ce_3_1_lg.py` can be similarly applied to this function as well.

Reply:
Addressed them with commit [https://github.com/jukent/GeoCAT-examples/commit/33aaeca2c09e8763983cb8ca1b7a4b26c4459a1c](url)

Reply:
NCL_ce_3_1_lg:

![image](https://user-images.githubusercontent.com/32553057/74766844-06e24500-5243-11ea-9ad3-4c4feb135397.png)

NCL_ce_3_2_lg:

![image](https://user-images.githubusercontent.com/32553057/74766897-21b4b980-5243-11ea-9806-8d4250dfac43.png)